networking/p2p/src/main/java/tech/pegasys/artemis/networking/p2p/jvmlibp2p/gossip/GossipMessageHandler.java

```diff
@@ -69,7 +84,35 @@ static PubsubPublisherApi createPublisher(final Gossip gossip, final PrivKey pri
       final EventBus eventBus,
       final ChainStorageClient chainStorageClient) {
     return List.of(
-        new AttestationTopicHandler(publisher, eventBus, chainStorageClient),
+        new AggregateTopicHandler(publisher, eventBus, chainStorageClient),
         new BlocksTopicHandler(publisher, eventBus, chainStorageClient));
   }
+
+  @Subscribe
+  public void registerAttestationTopicHandlers(CommitteeAssignmentEvent assignmentEvent) {
+    List<Integer> committeeIndices = assignmentEvent.getCommitteeIndices();
+    for (int i : committeeIndices) {
```

Comment:
nit: I'd be tempted to rename `i` to `index` or even `committeeIndex` to avoid it being confused with the index within the `committeeIndices` list.

Reply:
done.